README.md

```diff
@@ -71,7 +138,16 @@ Sample:
   }
 }
 ```
+In Curl it would be something like that:
+```
+curl -H 'Authorization: Token token="0b71145b7474d575632b"' -d "recipe[name]=Meatballzz" -d "recipe[difficulty]=2" -X PUT http://localhost:3000/recipes/22
+```
+
 
 ## Delete recipes
 
 `DELETE`: `/recipes/:id`
+In Curl it would be something like that:
+```
+curl -H 'Authorization: Token token="0b71145b7474d575632b"' -X DELETE http://localhost:3000/recipes/22
+```
```

Comment:
That's a good point. I think most people reading this would be looking to consume the API rather than develop on it, so I'd say _hyper-recipes.herokuapp.com_.